skbio/io/__init__.py

```diff
@@ -175,7 +184,13 @@
    `FASTQ documentation <https://scikit.bio/docs/dev/generated/skbio.io.format.fastq.html
    #examples>`_.
 
-As an example, to read a ``newick`` file using both interfaces you would type:
+As an example, let's read a
+`newick <https://scikit.bio/docs/latest/generated/skbio.io.format.newick.html>`_
```

Comment:
I guess this link can be replaced with

```
:mod:`format.newick <newick>`
```

If not working, add `skbio.io.` before `format`.

Can you give it a try?